### PR TITLE
Add kafka-connect dashboard

### PR DESCRIPTION
**Pull request by francoisferrand**
kafka-connect-1-general
kafka-connect-2-workers
kafka-connect-3-workers
kafka-connect-4-connectors
kafka-connect-5-rebalances
kafka-connect-6-tasks
kafka-connect-7-task-errors
kafka-connect-8-source
kafka-connect-9-sink
kafka-connect-10-system

Issue: ZENKO-4105

### CODE REVIEW

File: monitoring/kafka-connect/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,820 @@
...
+
+up = Stat(
+    title="Workers",
+    description="Number of workers available",
+    dataSource='${DS_PROMETHEUS}',
+    minValue='0',
+    maxValue='${replicas}',
+    reduceCalc='last',
+    targets=[Target(
+        expr='sum(up{job="${job}"})',
```
Comment (welansari) [inline monitoring/kafka-connect/dashboard.py]:
```suggestion
        expr='sum(up{job="${job}", namespace="${namespace}", instance="${instance}"})',
```

File: monitoring/kafka-connect/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,820 @@
...
+    legendDisplayMode='table',
+    legendPlacement='right',
+    unit=UNITS.SHORT,
+    targets=[
+        Target(
+            expr='sum(' + Metrics.worker_connector_running_task_count() + ')',
+            legendFormat='running'
+        ),
+        Target(
+            expr='sum(' + Metrics.worker_connector_running_task_count() + ')',
```
Comment (welansari) [inline monitoring/kafka-connect/dashboard.py]:
```suggestion
            expr='sum(' + Metrics.worker_connector_failed_task_count() + ')',
```

File: monitoring/kafka-connect/dashboard.py [hunk 3]
```diff
@@ -0,0 +1,820 @@
...
+    PieChart,
+    Stat,
+    Target,
+    TimeSeries,
+)
+
+
+class Metrics:
+    connector_status = metrics.Metric(
+        'kafka_connect_connector_status', 'status', 'instance', job='${job}'
```
Comment (welansari) [inline monitoring/kafka-connect/dashboard.py]:
```suggestion
        'kafka_connect_connector_status', 'status', 'instance', job='${job}', namespace='${namespace}'
```

same for the other metrics

File: monitoring/kafka-connect/dashboard.py (hunk 1 above)
Comment (francoisferrand) [reply monitoring/kafka-connect/dashboard.py]: Not sure we should filter on 'instance' here, as it would be displayed as a warning (yellow/red)...

**Review by welansari — APPROVED** [review]

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**